GUI.cpp

```diff
@@ -514,29 +507,23 @@ namespace
             0);
     }
 
-    void drawDebugCursor(WorldSegment* segment)
+    void drawSelectionCursor(WorldSegment* segment)
     {
         auto& ssConfig = stonesenseState.ssConfig;
+        Crd3D& selection = segment->segState.dfSelection;
+        if ((selection.x != -30000 && ssConfig.config.follow_DFcursor)
+            || (ssConfig.config.track_mode == Config::TRACKING_FOCUS)) {
```

Comment:
TRACKING_FOCUS is no longer a thing after https://github.com/DFHack/stonesense/pull/173